lib/configure.js

```diff
@@ -66,6 +68,13 @@ travis.users.sync.post = promisify(travis.users.sync.post);
 // when creating GitHub tokens.
 var noteUrl = 'https://github.com/mozilla/oghliner';
 
+// The message currently displayed with a spinner, if any.  Most async
+// operations don't need to set this, but occasionally one displays a message
+// and then calls a function that displays another message before
+// the operation is complete.  In that case, the function needs to stop
+// the spinner for the first message before it displays the second message.
+var spinnerMessage;
```

Comment:
16ddb6b completes this work, making the messages appear in the correct order:

```
✓ Deleting temporary GitHub token for getting Travis token… done!
✓ Creating permanent GitHub token for Travis to push to the repository… done!

ℹ You had an existing token for this app, so we deleted and recreated it.
```

However, while implementing it, I realized that there are two other circumstances under which we'll want to pause/interrupt the spinner to display another message:

First, on an authentication failure. A username/password failure isn't likely to occur now that we check their values before displaying a message with a spinner. But an auth code failure can still occur when the current auth code expires, and that can happen during any operation that uses the GitHub API and requires authentication:

```
You're using two-factor authentication with GitHub.
Please enter the code provided by your authentication software.

Auth Code: 123456

  Creating temporary GitHub token for getting Travis token…
Your authentication code is incorrect or has expired.
Please try again.

Auth Code: 987654
```

Second, when we tell Travis to sync with GitHub and then display `Waiting for Travis to finish syncing…` messages every five seconds:

```
I didn't find your repository in Travis.  Syncing Travis with GitHub…
Waiting for Travis to finish syncing…
Waiting for Travis to finish syncing…
```

For the latter case, we can solve the problem by making `Syncing Travis with GitHub…` a spinner message and then not printing `Waiting for Travis to finish syncing…`. We only display that message every five seconds now as a kind of poor-man's spinner, so the user knows that activity is ongoing, and the program isn't frozen. We can replace it with a real spinner.

But we should still have a plan for the former case, so we do something sensible when we need to pause/interrupt a spinner temporarily to request a new auth code.